cg/cli/upload/clinical_delivery.py

```diff
@@ -104,14 +105,24 @@ def auto_fastq(context: click.Context, dry_run: bool):
                 )
                 analysis_obj.uploaded_at = dt.datetime.now()
             else:
-                LOG.warning(
+                LOG.debug(
                     "Upload to clinical-delivery for %s has already started, skipping",
                     analysis_obj.family.internal_id,
                 )
             continue
         case: Family = analysis_obj.family
         LOG.info("Uploading family: %s", case.internal_id)
         analysis_obj.upload_started_at = dt.datetime.now()
-        context.invoke(upload_clinical_delivery, case_id=case.internal_id, dry_run=dry_run)
+        try:
+            context.invoke(upload_clinical_delivery, case_id=case.internal_id, dry_run=dry_run)
+        except Exception as error:
+            LOG.error(f"Upload of case {case.internal_id} failed")
+            LOG.error(error)
+            exit_code: int = EXIT_FAIL
+            continue
+
         if not dry_run:
             status_db.session.commit()
+
+        if exit_code:
```

Comment:
Im wondering if it should check for fail as `exit_code` is defined above as  `EXIT_SUCCESS`
```suggestion
        if exit_code == EXIT_FAIL:
```

Reply:
Any non zero exit code indicates an error occurred, but maybe it is better to be explicit

Reply:
Changed to `if exit_code != EXIT_SUCCESS:`